project/plugins.sbt

```diff
@@ -7,3 +7,7 @@ resolvers += Resolver.bintrayRepo("coursera", "sbt-plugins")
 
 // Courier binding generator plugin
 addSbtPlugin("org.coursera.courier" % "courier-sbt-plugin" % "2.0.2")
+
+addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "1.1")
+
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
```

Comment:
I'm curious: we seem to [only publish to Maven Central in courscala](https://github.com/coursera/courscala/blob/aeb45be956abd744eb49a59113595470a8d83594/scripts/publish-mavencentral). Are you going to update that repo too, to publish to Sonatype? I'm not really sure why we'd do one or the other.